pages/common/sdiff.md

```diff
@@ -0,0 +1,23 @@
+# sdiff
+
+> Compare the differences and/or merge between two files.
```

Comment:
Ah, looks like I missed the description What about this?
```suggestion
> Compare the differences between and optionally merge 2 files.
```

Reply:
Agree, I'll commit this suggestion.